operators/test-data/raster/readme.md

```diff
@@ -0,0 +1,9 @@
+# required test data
+
+## SRTM Tile(s)
+
+Get the SRTM tile 38_03 and unzip it into a subfolder "srtm_38_03".
+
+`wget http://srtm.csi.cgiar.org/wp-content/uploads/files/srtm_5x5/tiff/srtm_38_03.zip`
+
+`unzip -d srtm_38_03 srtm_38_03.zip`
```

Comment:
As stated before, for unit tests, I would commit test data into the repo.
Mapping has small test files for this case.
You could even replicate mapping test cases then to see if the behavior is similar.

Reply:
I will investigate if the mapping test data is working. I guess we could also generate some data with native tiling.